### DISCUSSION TITLE
GitHub Education benefits application's verification failed

### DISCUSSION THREAD

**Discussion by soumik-kanad** [community/community#146466] [GitHub Education]
### Hi everyone,

I am a PhD student and applied for continued Github education benefits after my first expiry. So, I had uploaded my transcript which has my name and current enrollment details and also has the university name at the top. But this application got rejected and  I'm not very sure why. The response is quite generic. 

So, can someone suggest what can be done here?
<img width="278" alt="image" src="https://github.com/user-attachments/assets/d130a2c9-d60b-4432-b53b-23233310147b">

**Comment by davevad93** [comment]
Hi @soumik-kanad ,

The application response suggests that you haven't completed your [billing information](https://github.com/settings/billing/payment_information) with your full name and also that you don't have `2FA` enabled. If you have already done so, sign out, then sign back in to GitHub and resubmit your request.
